invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/default_static_page.html

```diff
@@ -16,11 +16,12 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 #}
-{% extends config.PAGES_BASE_TEMPLATE %}
+{% extends "invenio_communities/base.html" %}
```

Comment:
move these modifications to zenodo_rdm/templates/themes/horizon/invenio_app_rdm/default_static_page.html